packages/firebase_dynamic_links/ios/Classes/FLTFirebaseDynamicLinksPlugin.m

```diff
@@ -118,63 +118,57 @@ - (NSMutableDictionary *)getInitialLink {
   return getDictionaryFromDynamicLink(_initialLink);
 }
 
-- (BOOL)application:(UIApplication *)application
-            openURL:(NSURL *)url
-            options:(NSDictionary<UIApplicationOpenURLOptionsKey, id> *)options {
-  return [self checkForDynamicLink:url];
-}
```

Comment:
info: nothing has changed, the method is moved lower where the rest of `-application: xxx` method are placed.